CHANGES_NEXT_RELEASE

```diff
@@ -1,3 +1,4 @@
+- Upgrade mongodb dependency to 2.2.35
```

Comment:
```suggestion
- Upgrade mongodb dependency from ~2.2.31 to ~2.2.35
```

Maybe over-detailed, but to keep the same style we used in the past for changelog...